src/Sarif.Multitool.FunctionalTests/TestData/UriMustBeAbsolute/Invalid.sarif

```diff
@@ -40,7 +40,7 @@
             "runs[0].tool.downloadUri",
             "runs[0].results[0].workItemUris[0]",
             "runs[0].originalUriBaseIds.SRCROOT",
-            "runs[0].resources.rules.TST0001.helpUri",
+            "runs[0].resources.rules[0].helpUri",
```

Comment:
>runs[0].resources.rules[0].helpUri [](start = 13, length = 34)

I hope you are enjoying the easy maintainability :-)

Reply:
indeed! much improved. :)

---
In reply to: [249139939](https://github.com/Microsoft/sarif-sdk/pull/1223#discussion_r249139939) [](ancestors = 249139939)